package.json

```diff
@@ -41,7 +41,7 @@
     "ajv": "^6.12.0",
     "cheerio": "^1.0.0-rc.3",
     "cheerio-tableparser": "^1.0.1",
-    "country-levels": "https://github.com/hyperknot/country-levels/releases/download/v1.2.0/export_q7.tgz",
+    "country-levels": "https://github.com/hyperknot/country-levels-releases/releases/download/v.1.4.0/export_q7.tgz",
```

Comment:
? is this legit?

Reply:
So legit

Reply:
Yes, having the releases in the main repo doesn't allow me to compress the repo with BFG Repo-Cleaner https://rtyley.github.io/bfg-repo-cleaner/